src/components/RequestsFilters/RequestsFilters.js

```diff
@@ -37,10 +40,12 @@ export default class RequestsFilters extends React.Component {
   };
 
   transformTagsOptions = () => {
-    return get(this.props.resources, 'tags.records', []).map(({ label }) => ({
-      label,
-      value: label,
-    }));
+    const tags = get(this.props.resources, 'tags.records', []);
+
+    return chain(tags)
```

Comment:
It would be great not to use chain, as it actually the wrapper on the whole lodash.
`_.chain = (array) => wrap(array, _); // Rough concept of chain`

So the `map` which is imported above is not actually used. Instead used the `.map` provided by a chain. Please read more information about it in the [article](https://medium.com/making-internets/why-using-chain-is-a-mistake-9bc1f80d51ba).